docs/install/mac/activating-a-validator.md

```diff
@@ -75,14 +74,14 @@ The beacon node will spin up and immediately begin communicating with the Prysm
 
 The process of syncronising may take a while; the incoming block per second capacity is dependent upon the connection strength, network congestion and overall peer count.
 
-## Starting up the validator client
+## Step 4b: Starting up the validator client
 
-**NOTICE:** The beacon node you are using should be **completely synced** before submitting your deposit for the validator client, otherwise the validator will not be able to validate and will **inflict minor inactivity balance penalties**.
+**NOTICE:** The beacon-chain node you are using should be **completely synced** before submitting your deposit. You may **incur minor inactivity balance penalties** if the validator is unable to perform its duties by the time the deposit is processed and activated by the ETH2 network.
 
 
-Open a second terminal window. Depending on your installation method, issue the appropriate command from the examples below to start the validator.
+Open a second terminal window. Depending on your platform, issue the appropriate command from the examples below to start the validator.
 
-#### Starting the validator client with Docker
+#### Starting the validator client with Docker on GNU/Linux or macOS
```

Comment:
Why include GNU/Linux note if it's a mac only page?

Reply:
Removed